pkg/chunk/schema_util.go

```diff
@@ -2,15 +2,22 @@ package chunk
 
 import (
 	"bytes"
+	"crypto/sha256"
 	"encoding/base64"
 	"encoding/binary"
 	"encoding/hex"
+	"encoding/json"
 
 	"fmt"
 
 	"github.com/prometheus/common/model"
 )
 
+func metricSeriesID(m model.Metric) string {
+	h := sha256.Sum256([]byte(m.String()))
+	return string(encodeBase64Bytes(h[:]))
```

Comment:
Out of curiosity, is there a special reason we use `base64` here vs., for example, `hex.EncodeToString` which:
- is more natural for such hash functions, and 
- could be more convenient to use when debugging? (or e.g. printing https://play.golang.org/p/IeHqPaaNk4)